eng/versioning/version_client.txt

```diff
@@ -29,6 +30,7 @@ com.azure:azure-storage-common;12.3.0;12.4.0-beta.1
 com.azure:azure-storage-file-share;12.1.0;12.2.0-beta.1
 com.azure:azure-storage-file-datalake;12.0.0-beta.10;12.0.0-beta.11
 com.azure:azure-storage-queue;12.2.0;12.3.0-beta.1
+com.azure:azure-storage-perf;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
Nit: Please put these in alphabetical order. azure-storage-perf above azure-storage-queue and if perf-test-core is common and not actually part of core, list that alphabetically in the file please.

Reply:
done